packages/web-components/src/components/rux-button/rux-button.scss

```diff
@@ -1,145 +1,117 @@
 :host {
-    display: inline-block;
-    width: auto;
-}
+    /* @TODO: Should these be moved into the design tokens library? */
+    /* references */
+    --disabled-opacity: var(--opacity-40);
+    --size: 0.25rem;
+
+    /* @TODO: Should these be moved into the design tokens library? */
+    /* aliases */
+    --button-font-size: var(--font-body-1-font-size);
+    --button-line-size: var(--line-height-sm);
 
-:host([hidden]) {
-    display: none;
+    cursor: pointer;
+    display: inline-flex;
+    flex-flow: column wrap;
 }
 
 :host([disabled]) {
-    pointer-events: none;
+    cursor: not-allowed;
 }
-.rux-button {
-    display: flex;
-    position: relative;
+
+button {
+    appearance: none;
+    background-color: transparent;
+    border-width: 0;
+    color: inherit;
+    font: inherit;
+    line-height: inherit;
     margin: 0;
-    width: inherit;
-    padding: 0 1rem;
-    height: 2.125rem;
-    min-width: 2.25rem;
+    padding: 0;
+}
+
+.rux-button {
+    align-items: center;
     border-radius: var(--radius-base);
-    color: var(--color-text-inverse);
-    font-family: var(--font-body-1-font-family);
-    font-size: var(--font-body-1-font-size);
-    font-weight: var(--font-body-1-font-weight);
-    letter-spacing: var(--font-body-1-letter-spacing);
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
+    cursor: pointer;
+    box-shadow: var(--border-color, transparent) 0px 0px var(--border-width, 0px) inset;
+    display: flex;
     justify-content: center;
-    align-items: center;
     user-select: none;
 
-    rux-icon,
-    ::slotted(rux-icon) {
```

Comment:
If its not too much work, I'd say go for it just to avoid having to make a breaking change. I'd say the 'one right way' would actually be the slot if we had to pick one but I don't have strong opinions

Reply:
This has been added back!